### ISSUE TITLE
font color in striped mode ignored on colored tabs

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#7]
When a font color is defined in "striped mode" this is not used on tabs that have a colored striped on top.

**Comment by RealRaven2000** [comment]
This appears to be caused by a syntax error using the` :not` pseudo selector in `quickfolders-interface.js`, line 6309:

```
  engine.setElementStyle(ss, '.quickfolders-flat toolbarbutton:not[#QuickFoldersCurrentFolder]' + coloredPaletteClass + ' > label','color', inactiveColor, false);
  engine.setElementStyle(ss, '.quickfolders-flat toolbarbutton:not[#QuickFoldersCurrentFolder]' + paletteClass + ' > label','color', inactiveColor, false);
```

**Comment by RealRaven2000** [comment]
Here is a tryout version (4.17 prerelease 25) with the bugfix:

[QuickFolders-wx-4.17pre25.zip](https://github.com/RealRaven2000/QuickFolders/files/3696776/QuickFolders-wx-4.17pre25.zip)

**Comment by crti** [comment]
Not sure if the usage of DarkMode Theme and this white on bright tabs is related to the issue: 
![QuickFolders-DarkMode](https://user-images.githubusercontent.com/1004113/68303798-de27ac80-00a4-11ea-91fb-6f3c7182f1a9.png)

**Comment by RealRaven2000** [comment]
Did you try pre25 ? Also are you using Flat Style  +Striped? Can't really see any "stripe" colors on your screenshot... maybe a screenshot of your QuickFolders "Layout / Bling my tabs" settings (third tab, after Advanced) would be helpful.

**Comment by crti** [comment]
Thx for the hint. Sorry for not trying myself in the QF settings. I'm now on Apple Pills which I like the most so far.

**Comment by RealRaven2000** [comment]
Fixed in 4.17.2
